addons/l10n_it_edi/models/l10n_it_edi_export.py

```diff
@@ -0,0 +1,88 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
... I will comply... because I know Odoo it's just like that
But files with thousands of lines and sections are awful.
It's harder to modify them, it makes refactoring and moving functions a hell of diffs. and you have to scroll it all to understand what happens. 
There are files and classes in programming, huh. You don't need to put everything in the same file.